extensions/amp-carousel/0.1/build-dom.js

```diff
@@ -229,6 +231,8 @@ function buildSlideScrollCarousel(element) {
     slidesContainer.appendChild(slideWrapper);
     slideWrappers.push(slideWrapper);
   });
+  // Initialize the first slide to be shown.
+  slideWrappers[0]?.classList.add(ClassNames.SLIDES_ITEM_SHOW);
```

Comment:
Nit: Move this into the `forEach` above? Also, why isn't it a `map`…

Reply:
I'm confused. Why should this be part of the forEach and what should be a map?

Reply:
Oh I see, instead of arr.forEach it could be arr.map.
Mind if I punt on that to a small PR tomorrow?